components/proxy/src/test/java/com/hotels/styx/routing/StaticPipelineBuilderTest.java

```diff
@@ -74,7 +80,7 @@ public void appliesPluginsInOrderTheyAreConfigured() throws Exception {
                 interceptor("Test-B", appendResponseHeader("X-From-Plugin", "B"))
         );
 
-        HttpHandler handler = new StaticPipelineFactory(clientFactory, environment, registry, plugins, false).build();
+        HttpHandler handler = new StaticPipelineFactory(clientFactory, environment, registry, plugins, eventLoopGroup, socketChannelClass,false).build();
```

Comment:
```suggestion
        HttpHandler handler = new StaticPipelineFactory(clientFactory, environment, registry, plugins, eventLoopGroup, socketChannelClass, false).build();
```
Missing space is probably a checkstyle violation.